Modify rule S3973: LaYC Indenting if (#1947)
SonarSource/rspec: rules/S3973/csharp/rule.adoc

include::../description.adoc[]

[source,csharp]
----
if (condition)  // Noncompliant
DoTheThing();

> DoTheOtherThing(); // Did we really intend to always call this function?

Reviewer: Avoid using "we"
```suggestion
DoTheOtherThing(); // Was the intend to call this function unconditionally?
```

Author: Done

----

Indentation makes the intent clear.

[source,csharp]
----
if (condition)
  DoTheThing();

> DoTheOtherThing(); // We clearly intended to always call this function.

Reviewer: ```suggestion
DoTheOtherThing(); // A clear intent to call this function unconditionally.
```

Author: Done
